packages/driver/cypress/e2e/e2e/origin/origin.cy.ts

```diff
@@ -224,7 +224,9 @@ describe('cy.origin', () => {
     describe('errors', () => {
       it('propagates secondary origin errors to the primary that occur within the test', (done) => {
         cy.on('fail', (err) => {
-          expect(err.message).to.include('variable is not defined')
+          const undefinedMessage = Cypress.isBrowser('webkit') ? 'Can\'t find variable: variable' : 'variable is not defined'
```

Comment:
while this isn't used right now, it will be after we enable cy.origin for webkit.